Lib/test/test_grammar.py

```diff
@@ -1099,6 +1100,14 @@ def testAssert2(self):
         else:
             self.fail("AssertionError not raised by 'assert False'")
 
+        with self.assertWarnsRegex(SyntaxWarning, 'assertion is always true'):
+            compile('assert(x, "msg")', '<testcase>', 'exec')
+        with warnings.catch_warnings():
+            warnings.filterwarnings('error', category=SyntaxWarning)
+            with self.assertRaisesRegex(SyntaxError, 'assertion is always true'):
+                compile('assert(x, "msg")', '<testcase>', 'exec')
+            compile('assert x, "msg"', '<testcase>', 'exec')
```

Comment:
Whoo, `compile('assert(x, "msg")')` raise `SyntaxWarning` while `compile('assert(1, "msg")')` not. Interesting ast optimization.

Reply:
It is possible to add a support for a constant tuple, but I think this is not worth.

Reply:
Agreed.